### ISSUE TITLE
Cyberpunk 2077 Debugging Problem

### ISSUE THREAD

**Issue by Vegeta780** [CyB1K/PS4-Fake-PKG-Tools-3.87#8]
Hi Greetings sir 
i have a issue with 
Cyberpunk 2077  CUSA16579 – EUR (@Arczi) Update 1.31 (Fix 5.05/6.72/7.02) (Phat /Slim)
Cyberpunk 2077 CUSA18278 – EUR (@Opoisso893) @Golemnight/Opoisso893 Update 1.31 (5.05/6.72/7.02) 
i tried both of them games Games working fine but i can't debug or use ps4cheater apps  if i use game crash 
Developer of ps4cheaters said, there is problem with  Dump game 
debugging options working fine with duplex release but there is no back port of Duplex release 
is there any  solution ?
Best Regards

**Comment by CyB1K** [comment]
I dont remember which one i did, but thats unrelated with the tools

**Comment by Vegeta780** [comment]
Yes I know But Its a request can you back port Duplex release and  Nobody care  about debugging option either 
its a specials request i know only you can backport so any chance you can do ? thanks
Best Regards
